src/openforms/formio/components/custom.py

```diff
@@ -457,18 +481,23 @@ def validate_postcode(self, value: str) -> str:
     def validate(self, attrs):
         attrs = super().validate(attrs)
 
+        auto_populated = attrs.get("autoPopulated", False)
+        postcode = attrs.get("postcode", "")
+        house_number = attrs.get("houseNumber", "")
         city = attrs.get("city", "")
         street_name = attrs.get("streetName", "")
 
         if self.derive_address:
-            existing_hmac = attrs.get("secretStreetCity", "")
-            postcode = attrs.get("postcode", "")
-            number = attrs.get("houseNumber", "")
+            # when the user fills in manually the city and the street name we do not
+            # need to check the secret city - street name combination
+            if not auto_populated:
+                return attrs
```

Comment:
ah okay, and because `autoPopulated = False` we then also know in the registration backend that the data can potentially not be trusted, right?

Reply:
Yes, this is how I thought about it.Let me know if you agree.